.github/workflows/homebrew.yml

```
@@ -0,0 +1,17 @@
+name: homebrew
+
+on:
+  push:
+    tags: 'v*'
+
+jobs:
+  homebrew:
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v1
+        with:
+          # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula:
+          formula-name: checkov
+        env:
+          COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
```

Thanks @chenrui333 !

We do have a token already in our secrets, can we reuse it?
It's in use in: https://github.com/bridgecrewio/checkov/blob/52c7a73f9788a88a7f90408309c5fb6fa827f4d5/.github/workflows/build.yml#L176
```suggestion
          COMMITTER_TOKEN: ${{ secrets.PAT_TOKEN }}
```

---

Nice!! that is what I want.